documentation/book/appendix_crds.adoc

```diff
@@ -443,10 +443,12 @@ Used in: xref:type-NodePortListenerOverride-{context}[`NodePortListenerOverride`
 
 [options="header"]
 |====
-|Property         |Description
-|address   1.2+<.<|Additional address name for the bootstrap service. The address will be added to the list of subject alternative names of the TLS certificates.
+|Property               |Description
+|address         1.2+<.<|Additional address name for the bootstrap service. The address will be added to the list of subject alternative names of the TLS certificates.
 |string
-|nodePort  1.2+<.<|Node port for the bootstrap service.
+|dnsAnnotations  1.2+<.<|Annotations which will be added to the Service resource. You can use this field to instrument DNS providers such as External DNS.
```

Comment:
Please amend:

"Annotations which will be added" > "Annotations that will be added"

Service in monospace? `Service`

"Instrument" is a rather obscure verb. Can we use "configure" or similar instead?

Reply:
Thanks for the comment. I fixed it here as well as in the other files which used the same formulation.